package.json

```diff
@@ -1,31 +1,36 @@
 {
   "scripts": {
-    "build": "./node_modules/.bin/webpack",
-    "dev": "./node_modules/.bin/webpack-dev-server --port 3000 --hot",
+    "build": "./node_modules/.bin/webpack --config webpack.config.build.js",
+    "dev": "./node_modules/.bin/webpack-dev-server --config webpack.config.dev.js --port 3000 --hot",
     "format": "./node_modules/.bin/elm-format",
     "start" : "node server.js",
     "test": "./node_modules/.bin/elm-test"
   },
   "devDependencies": {
-    "babel-core": "^6.26.0",
-    "css-loader": "^0.28.7",
-    "elm": "^0.18.0",
-    "elm-assets-loader": "^0.3.0",
-    "elm-format": "0.6.1-alpha",
-    "elm-test": "0.18.12",
-    "elm-webpack-loader": "^4.4.0",
-    "file-loader": "^1.1.5",
-    "node-sass": "^4.6.1",
+    "autoprefixer": "^9.4.4",
+    "babel-core": "^6.26.3",
+    "copy-webpack-plugin": "^4.6.0",
+    "css-loader": "^2.0.0",
+    "elm": "^0.19.0",
+    "elm-test": "^0.19.0",
```

Comment:
Should we have `elm-format` as a dependency here?

Reply:
Yes. I think so.. Good spot!

Reply:
Done.